README.md

```diff
@@ -2,13 +2,13 @@
 
 ![Medplum](packages/docs/static/img/cover.webp)
 
-Medplum is a developer platform that enables flexible and rapid development of healthcare apps.
+Medplum is a developer platform that enables the flexible and rapid development of healthcare apps.
```

Comment:
This change is unnecessary, and doesn't improve the clarity of the text; please revert it